azure-pipelines.yml

```diff
@@ -37,6 +37,54 @@ stages:
               failTaskOnFailedTests: true
             condition: always()
 
+  - stage: PlanResponse
```

Comment:
Could we use parameters here and have a for_each instead?

Reply:
yeah sounds good will correct :)